AnkiDroid/src/main/java/com/ichi2/libanki/Utils.java

```diff
@@ -195,6 +195,9 @@ public static String remainingTime(Context context, long time_s) {
         Resources res = context.getResources();
         if (time_s < TIME_HOUR_LONG) {
             time_x = (int) Math.round(time_s / TIME_MINUTE);
+            if (time_x == 0) {
+                time_x++;
+            }
```

Comment:
This is going to seem like a nitpick but I think it's important

1- if the idea is to "clamp" something, you want to wipe out the whole range below the value you want to clamp, just in case, not just one specific value (0), so Math.max(ETA, 1) is better (changing it to `< 1` as a comparison could also work but we're already using Math methods, just throw one more in...)
2- there should be a comment so future code readers know why we are doing this

```suggestion
            // get time remaining, but never less than 1
            time_x = Math.max((int) Math.round(time_s / TIME_MINUTE), 1);
```

Reply:
Thanks for the suggestions
Definitely gonna keep them in mind the next time I contribute